### ISSUE TITLE
no template could be found with the group identity

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/wpf#448]
VSF_TYPE_MARKDOWNI have installed the visual studio community 2019 preview for the first time. When i tried to create a new wpf project with .net core it gave error &quot;no template could be found with the group identity&quot;. In parallel i also have vs community 2017.

_This issue has been moved from https://developercommunity.visualstudio.com/content/problem/495671/no-template-could-be-found-with-the-group-identity-2.html
VSTS ticketId: 822402_
_These are the original issue comments:_

zubairrana07 on 3/18/2019, 01:47 PM (2 hours ago): no template could be found with the group identity "Microsoft.Common.Wpf"
_These are the original issue solutions:_
(no solutions)

**Comment by zubairrana** [comment]
Thank you for creating new issue here against my report. The solution explained at another link does not work as when i opened the setting the preview option was already checked. 

https://developercommunity.visualstudio.com/content/problem/473540/vs-2019-rc-cant-create-winforms-net-core-multiple.html 

Against this issue on developercommunity i had submitted the video recording using the visual studio recording option which automatically created some log files and were attached with the report. I was told the attachments (video + log) will be private to microsoft only that's why they are not visible here.

**Comment by grubioe** [comment]
@zubairrana is this still an issue with the latest builds and VS 2019 previews?

**Comment by zubairrana** [comment]
@grubioe Now i am using VS 2019 and .net core 3.0 preview is working on it for WPF.

**Comment by grubioe** [comment]
Great, thanks for confirming!
